linode/manager
# refactor: [M3-6943] - MUI v5 Migration - `SRC > Components > InputAdornment`

## Description 📝

- Adjusts `InputAdornment` to match the import/export structure of our other components

## Major Changes 🔄
- Use named exports 
- Document in Storybook 📝
- Move out of the `core` folder 🗂️

## Preview 📷
![Screenshot 2023-07-26 at 6 54 58 PM](https://github.com/linode/manager/assets/115251059/e62dc303-8847-41e6-bc44-8472b2f6a92d)

## How to test 🧪
- Verify that input adornments are unaffected by this change
  -  One place you can see them is in the "Make a Payment Drawer" (http://localhost:3000/account/billing/make-payment)
